player/src/bin/play.rs

```diff
@@ -17,7 +17,13 @@ fn main() {
     #[cfg(feature = "winit")]
     use winit::{event_loop::EventLoop, window::WindowBuilder};
 
-    env_logger::init();
+    #[cfg(feature = "subscriber")]
```

Comment:
Player doesn't have this feature. Since it's a debugging tool, we should probably just enable it in `wgc` dependency for the player. But the `WGPU_CHROME_TRACE` is probably not the best way to tell the player where to put the trace. I think, ideally, we'd want some `--trace <path>` parameter on command line? Anyway, we can follow-up with this, not a blocker.

Reply:
We should probably grab one of the mini arg parsing libs to use for player, I'm personally partial to preco-args, but doesn't ultimately matter.